src/kms-message/src/kms_port.c

```diff
@@ -15,8 +15,13 @@
  */
 
 #include "kms_port.h"
+
+#include <stdlib.h>
+#include <string.h>
+
 #if defined(_WIN32)
-char * kms_strndup (const char *src, size_t len)
+char *
+kms_strndup (const char *src, size_t len)
```

Comment:
`src/kms-message` is a vendored copy from libmongocrypt.

Should the new warnings only be applied to `src/libbson`, `src/libmongoc`, and `src/common`?